deploy/scripts/src/cri-kiwi/test.ts

```diff
@@ -717,3 +761,37 @@ function randomDate (start: Date, end: Date): Date {
 function getCSRF (r: Response): string {
   return r.html().find("input[name='x-csrf-token']").val() ?? ''
 }
+
+function getRequestCode (r: Response): string {
+  const responseBody = r.body
+
+  if (responseBody !== null && typeof responseBody === 'string') {
+    const response = JSON.parse(responseBody)
+
+    if (response?.request !== null) {
+      const request = response.request
+      return request
+    } else {
+      return 'No value found'
+    }
+  } else {
+    return 'Invalid response body'
+  }
+}
+
+function getClientID (r: Response): string {
+  const responseBody = r.body
+
+  if (responseBody !== null && typeof responseBody === 'string') {
+    const response = JSON.parse(responseBody)
+
+    if (response?.clientId != null) {
+      const clientId = response.clientId
+      return clientId
+    } else {
+      return 'No value found'
+    }
+  } else {
+    return 'Invalid response body'
+  }
+}
```

Comment:
```suggestion
function getClientID (r: Response): string {
  const clientId = r.json('clientId')
  if (clientId !== null && typeof clientId === 'string') return clientId
  fail('Client ID not found')
}
```
This can be simplified using `Response.json()` built in function. Also we should call `fail()` in the case of unrecoverable errors, returning string values will cause the script to continue on.